Makefile.am

```diff
@@ -80,7 +80,8 @@ include_cpputest_HEADERS = \
 	include/CppUTest/TestResult.h \
 	include/CppUTest/TestTestingFixture.h \
 	include/CppUTest/Utest.h \
-	include/CppUTest/UtestMacros.h
+	include/CppUTest/UtestMacros.h \
+	CppUTestGeneratedConfig.h
```

Comment:
It gets installed in include/CppUTest. The file is the source file and that one is in the local directory because it is a generated file.